env/initialize.sh

```diff
@@ -14,9 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This scripts initializes the build environment, by downloading and setting up
+# all necessary tools and libraries.
+#
+# By default, existing directories and files are left intact (except the
+# "./activate" file), so it's safe to run this script multiple times. Add the
+# "-f" flag in order to enforce overwriting the existing state.
+
 
 set -eu
 
+is_forced=0
```

Comment:
nit: This name doesn't really convey what is enforced - maybe rename it to force_reinitialization or so

Reply:
Done